charts/grafaml/templates/configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
 
 {{- define "dashboardYaml" -}}
 title: "{{ .Values.title }}"
-uid: "{{ include "grafaml.fullname" . }}"
+uid: "{{ include "grafaml.uid" . }}"
```

Comment:
I tested this with a locally built chart from the feature branch. Grafana didn't cause any issues. Additionally I tried to import this using JSON in the Grafana UI and it recognizes that the Dashboard with the same UID (the SHA-1 sum here) already exists. I think this works as we intended it to. The URL was also adapted to reflect the new UID

Reply:
Sorry, my comment was a bit unclear. 
What i mean is, what happens if an old existing dashboard, using the current version of this helm chart, updates to the new version? I guess the URL of the existing dashboard will change. If this is the case, it would break any documented link the user has to the Dashboard. And this should be mentioned in the Changenotes

Reply:
This is what I meant, I updated a dashboard running with 2.0.1. to the new chart version. You are right that the URL for existing dashboards does change when updating to the new version.